android/assets/jsons/Civ V - Vanilla/UnitPromotions.json

```diff
@@ -144,13 +146,14 @@
 	{
 		"name": "Medic",
 		"prerequisites": ["Shock I", "Drill I", "Scouting II"],
-		"effect": "This unit and all others in adjacent tiles heal 5 additional HP per turn",
+		"uniques": ["+[5] HP when healing", "All adjacent units heal [5] additional HP when healing"],
 		"unitTypes": ["Melee","Mounted","Scout"]
 	},
 	{
 		"name": "Medic II",
 		"prerequisites": ["Medic"],
-		"effect": "This unit and all others in adjacent tiles heal 5 additional HP. This unit heals 5 additional HP outside of friendly territory.",
+		"uniques": ["+[5] HP when healing", "All adjacent units heal [5] additional HP when healing",
+					"+[5] HP when healing in [Foreign Land] tiles"],
```

Comment:
Same note, that way you could have units who are stronger but e.g. can't heal fast